great-idea-website/css/index.css

```diff
@@ -63,4 +63,74 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	max-width: 960px;
+}
```

Comment:
try to add inside your container class:
margin: 0 auto;

and check what happens ;)